packages/core/src/common/classes.ts

```diff
@@ -24,6 +24,7 @@ export const FIXED = "pt-fixed";
 export const FIXED_TOP = "pt-fixed-top";
 export const VERTICAL = "pt-vertical";
 export const ROUND = "pt-round";
+export const TEXT_OVERFLOW_ELLIPSIS = "pt-text-overflow-ellipsis";
```

Comment:
It would be useful to get autocomplete for the other ones in the typography. If we're doing one, no reason not to do the others too if we're maintaining this file